namerd/storage/zk/src/main/scala/com/twitter/finagle/serverset2/ZkDtabStore.scala

```diff
@@ -44,6 +44,22 @@ class ZkDtabStore(
     children <- zk.getChildrenOf(zkPrefix)
   } yield children.children.toSet
 
+  private[this] def ensurePath(zkw: ZooKeeperWriter, path: String) {
+    val components = Path.read(path)
+    if (components.size > 1) {
+      ensurePath(zkw, components.take(components.size - 1).show)
+    }
+
+    zkw.create(
+      path,
+      None,
+      Seq(Data.ACL.AnyoneAllUnsafe),
```

Comment:
not a blocker, this is obviously not production-ready.  Creating files with global-writable is dangerous for any real usage.  We'll have to revisit permissions and privileges before we tell anyone to use this outside of experimental use.  Maybe worth putting a note in the docs to this effect.